use_cases/details/navigator-ft.md

```diff
@@ -0,0 +1,14 @@
+![Generate synthetic tabular, text and time series data](https://blueprints.gretel.cloud/use_cases/images/navigator-ft-hero.png "Generate synthetic tabular, text and time series data")
+
+We are excited to announce the public preview of **Navigator fine-tuning**, the latest advancement in our suite of synthetic data solutions. This new feature builds upon the recent general availability of [Gretel Navigator](https://console.gretel.ai/navigator), enabling you to generate data not only from a prompt, but also from fine-tuning the underlying model on your domain-specific real-world datasets to generate the highest quality synthetic data.
```

Comment:
> We are excited to announce the public preview of **Navigator fine-tuning**

Should "fine-tuning" be capitalized? Also on line 5.

Reply:
@sami-t ? I copied that from the blog post.

Reply:
Let's go with Navigator Fine Tuning (capitalized and without the dash). The blog post draft is already updated. Thanks!